--- base
+++ state
@@ -1,5 +1,5 @@
 <body style="background-color: #E4E4E7">
 <div
-      data-pencil-name="Proyek baru / Terisi / Mobile"
+      data-pencil-name="Proyek baru / Menyimpan / Mobile"
       class="box-border w-[390px] h-[1864px] flex flex-col gap-0 justify-start items-start bg-[#E4E4E7] [outline:1px_solid_#F0F0F2] rounded-[36px] overflow-hidden"
     >
@@ -841,5 +841,5 @@
 <div
             data-pencil-name="Simpan draf"
-            class="box-border [flex:1_1_0] h-fit flex flex-row gap-[8px] p-[12px_36px] justify-center items-center bg-[#FFFFFF] [border:1px_solid_#F0F0F2] rounded-[999px]"
+            class="box-border [flex:1_1_0] h-fit opacity-[0.4] flex flex-row gap-[8px] p-[12px_36px] justify-center items-center bg-[#FFFFFF] [border:1px_solid_#F0F0F2] rounded-[999px]"
           >
 <div
@@ -852,11 +852,21 @@
 <div
             data-pencil-name="Buat proyek"
-            class="box-border [flex:1_1_0] h-fit flex flex-row gap-[8px] p-[12px_36px] justify-center items-center bg-[#2F5BFF] rounded-[999px]"
-          >
+            class="box-border [flex:1_1_0] h-fit opacity-[0.4] flex flex-row gap-[8px] p-[12px_36px] justify-center items-center bg-[#2F5BFF] rounded-[999px]"
+          >
+<svg
+              data-pencil-name="Pending indicator"
+              data-icon-name="loader-circle"
+              data-icon-set="lucide"
+              viewBox="0 0 13.99993896484375 14"
+              preserveAspectRatio="xMidYMid meet"
+              xmlns="http://www.w3.org/2000/svg"
+              class="box-border w-[16px] shrink-0 h-[16px]"
+            >
+</svg>
 <div
               data-pencil-name="Label"
               class="text-[14px]/[normal] box-border text-[#FFFFFF] font-['Plus_Jakarta_Sans',system-ui,sans-serif] font-semibold text-left [white-space:nowrap]"
             >
-              Buat proyek
+              Membuat proyek…
             </div>
 </div>
