--- base
+++ state
@@ -1,10 +1,10 @@
 <body style="background-color: #E4E4E7">
 <div
-      data-pencil-name="Tim / Anggota / Aktif / Desktop"
-      class="box-border w-[1440px] h-[1024px] flex flex-row gap-0 p-[12px_12px_12px_0px] justify-start items-start bg-[#E4E4E7] overflow-hidden"
+      data-pencil-name="Tim / Form anggota / Tambah / Desktop"
+      class="box-border w-[1440px] h-[1024px] flex flex-row gap-0 p-[12px_12px_12px_0px] justify-start items-start bg-[#E4E4E7] overflow-hidden relative"
     >
 <div
         data-pencil-name="Sidebar"
-        class="box-border w-[252px] shrink-0 h-full flex flex-col gap-[12px] p-[8px_12px] justify-start items-start bg-[#E4E4E7]"
+        class="box-border w-[252px] shrink-0 h-full flex flex-col gap-[12px] p-[8px_12px] justify-start items-start bg-[#E4E4E7] relative [z-index:0]"
       >
 <div
@@ -361,5 +361,5 @@
 <div
         data-pencil-name="Panel"
-        class="box-border [flex:1_1_0] h-full flex flex-col gap-0 justify-start items-start bg-[#F4F4F5] [border:1px_solid_#DCDCE0] rounded-[16px] overflow-hidden"
+        class="box-border [flex:1_1_0] h-full flex flex-col gap-0 justify-start items-start bg-[#F4F4F5] [border:1px_solid_#DCDCE0] rounded-[16px] overflow-hidden relative [z-index:1]"
       >
 <div
@@ -1395,4 +1395,254 @@
 </div>
 </div>
+<div
+        data-pencil-name="Overlay"
+        class="box-border w-[1440px] h-[1024px] absolute left-0 top-0 flex flex-row gap-0 justify-center items-center bg-[#09090B80] [z-index:2]"
+      >
+<div
+          data-pencil-name="Modal"
+          class="box-border w-fit shrink-0 h-fit flex flex-row gap-0 justify-center items-center"
+        >
+<div
+            data-pencil-name="Dialog"
+            class="box-border w-[560px] shrink-0 h-fit [box-shadow:0px_16px_40px_#18181B24] flex flex-col gap-0 justify-start items-start bg-[#FFFFFF] [border:1px_solid_#F0F0F2] rounded-[16px] overflow-hidden"
+          >
+<div
+              data-pencil-name="Header"
+              class="[box-sizing:content-box] w-[560px] h-[86.5px] shrink-0 flex flex-row gap-[12px] p-[20px_24px] justify-start items-start [border-width:0px_0px_1px_0px] [border-style:solid] [border-color:#F0F0F2] [margin:0px_0px_-0.5px_0px]"
+            >
+<div
+                data-pencil-name="Heading"
+                class="box-border [flex:1_1_0] h-fit flex flex-col gap-[4px] justify-start items-start"
+              >
+<div
+                  data-pencil-name="Title"
+                  class="text-[18px]/[normal] box-border text-[#18181B] font-['Plus_Jakarta_Sans',system-ui,sans-serif] font-bold tracking-[-0.4px] text-left [white-space:nowrap]"
+                >
+                  Tambah anggota
+                </div>
+<div
+                  data-pencil-name="Description"
+                  class="text-[13px]/[20px] box-border w-full text-[#52525B] font-['Plus_Jakarta_Sans',system-ui,sans-serif] font-normal text-left"
+                >
+                  Freelancer yang bisa kamu tugaskan di jadwal proyek.
+                </div>
+</div>
+<div
+                data-pencil-name="Close"
+                class="box-border w-fit shrink-0 h-fit flex flex-row gap-0 p-[8px] justify-center items-center rounded-[8px]"
+              >
+<svg
+                  data-pencil-name="Icon"
+                  data-icon-name="x"
+                  data-icon-set="lucide"
+                  viewBox="0 0 13.99993896484375 14"
+                  preserveAspectRatio="xMidYMid meet"
+                  xmlns="http://www.w3.org/2000/svg"
+                  class="box-border w-[16px] shrink-0 h-[16px]"
+                >
+</svg>
+</div>
+</div>
+<div
+              data-pencil-name="Body"
+              class="box-border w-full h-fit shrink-0 flex flex-col gap-[16px] p-[24px] justify-start items-start"
+            >
+<div
+                data-pencil-name="Nama"
+                class="box-border w-full h-fit shrink-0 flex flex-col gap-[6px] justify-start items-start"
+              >
+<div
+                  data-pencil-name="Label row"
+                  class="box-border w-full h-fit shrink-0 flex flex-row gap-[4px] justify-start items-center"
+                >
+<div
+                    data-pencil-name="Label"
+                    class="text-[12px]/[normal] box-border text-[#18181B] font-['Plus_Jakarta_Sans',system-ui,sans-serif] font-semibold text-left [white-space:nowrap]"
+                  >
+                    Nama
+                  </div>
+</div>
+<div
+                  data-pencil-name="Input"
+                  class="box-border w-full h-[40px] shrink-0 flex flex-row gap-[8px] p-[0px_12px] justify-start items-center bg-[#FFFFFF] [border:1px_solid_#DCDCE0] rounded-[8px]"
+                >
+<div
+                    data-pencil-name="Placeholder"
+                    class="text-[14px]/[normal] box-border [flex:1_1_0] text-[#71717A] font-['Plus_Jakarta_Sans',system-ui,sans-serif] font-normal text-left"
+                  >
+                    Nama lengkap
+                  </div>
+</div>
+<div
+                  data-pencil-name="Message"
+                  class="box-border w-full h-[17px] shrink-0 flex flex-row gap-[8px] justify-start items-center"
+                >
+</div>
+</div>
+<div
+                data-pencil-name="Nomor WhatsApp"
+                class="box-border w-full h-fit shrink-0 flex flex-col gap-[6px] justify-start items-start"
+              >
+<div
+                  data-pencil-name="Label row"
+                  class="box-border w-full h-fit shrink-0 flex flex-row gap-[4px] justify-start items-center"
+                >
+<div
+                    data-pencil-name="Label"
+                    class="text-[12px]/[normal] box-border text-[#18181B] font-['Plus_Jakarta_Sans',system-ui,sans-serif] font-semibold text-left [white-space:nowrap]"
+                  >
+                    Nomor WhatsApp
+                  </div>
+</div>
+<div
+                  data-pencil-name="Input"
+                  class="box-border w-full h-[40px] shrink-0 flex flex-row gap-[8px] p-[0px_12px] justify-start items-center bg-[#FFFFFF] [border:1px_solid_#DCDCE0] rounded-[8px]"
+                >
+<div
+                    data-pencil-name="Placeholder"
+                    class="text-[14px]/[normal] box-border [flex:1_1_0] text-[#71717A] font-['Plus_Jakarta_Sans',system-ui,sans-serif] font-normal text-left"
+                  >
+                    0812 3456 7890
+                  </div>
+</div>
+<div
+                  data-pencil-name="Message"
+                  class="box-border w-full h-fit shrink-0 flex flex-row gap-[8px] justify-start items-center"
+                >
+<div
+                    data-pencil-name="Helper"
+                    class="text-[12px]/[17px] box-border [flex:1_1_0] text-[#71717A] font-['Plus_Jakarta_Sans',system-ui,sans-serif] font-normal text-left"
+                  >
+                    Contoh: 0812 3456 7890
+                  </div>
+</div>
+</div>
+<div
+                data-pencil-name="Email"
+                class="box-border w-full h-fit shrink-0 flex flex-col gap-[6px] justify-start items-start"
+              >
+<div
+                  data-pencil-name="Label row"
+                  class="box-border w-full h-fit shrink-0 flex flex-row gap-[4px] justify-start items-center"
+                >
+<div
+                    data-pencil-name="Label"
+                    class="text-[12px]/[normal] box-border text-[#18181B] font-['Plus_Jakarta_Sans',system-ui,sans-serif] font-semibold text-left [white-space:nowrap]"
+                  >
+                    Email
+                  </div>
+<div
+                    data-pencil-name="Optional"
+                    class="text-[12px]/[normal] box-border text-[#71717A] font-['Plus_Jakarta_Sans',system-ui,sans-serif] font-normal text-left [white-space:nowrap]"
+                  >
+                    (opsional)
+                  </div>
+</div>
+<div
+                  data-pencil-name="Input"
+                  class="box-border w-full h-[40px] shrink-0 flex flex-row gap-[8px] p-[0px_12px] justify-start items-center bg-[#FFFFFF] [border:1px_solid_#DCDCE0] rounded-[8px]"
+                >
+<div
+                    data-pencil-name="Placeholder"
+                    class="text-[14px]/[normal] box-border [flex:1_1_0] text-[#71717A] font-['Plus_Jakarta_Sans',system-ui,sans-serif] font-normal text-left"
+                  >
+                    nama@email.com
+                  </div>
+</div>
+<div
+                  data-pencil-name="Message"
+                  class="box-border w-full h-[17px] shrink-0 flex flex-row gap-[8px] justify-start items-center"
+                >
+</div>
+</div>
+<div
+                data-pencil-name="Peran"
+                class="box-border w-full h-fit shrink-0 flex flex-col gap-0 justify-start items-start"
+              >
+<div
+                  data-pencil-name="Field"
+                  class="box-border w-full h-fit shrink-0 flex flex-col gap-[6px] justify-start items-start"
+                >
+<div
+                    data-pencil-name="Label row"
+                    class="box-border w-full h-fit shrink-0 flex flex-row gap-[4px] justify-start items-center"
+                  >
+<div
+                      data-pencil-name="Label"
+                      class="text-[12px]/[normal] box-border text-[#18181B] font-['Plus_Jakarta_Sans',system-ui,sans-serif] font-semibold text-left [white-space:nowrap]"
+                    >
+                      Peran
+                    </div>
+</div>
+<div
+                    data-pencil-name="Input"
+                    class="box-border w-full h-[40px] shrink-0 flex flex-row gap-[8px] p-[0px_12px] justify-start items-center bg-[#FFFFFF] [border:1px_solid_#DCDCE0] rounded-[8px]"
+                  >
+<div
+                      data-pencil-name="Placeholder"
+                      class="text-[14px]/[normal] box-border [flex:1_1_0] text-[#71717A] font-['Plus_Jakarta_Sans',system-ui,sans-serif] font-normal text-left"
+                    >
+                      Pilih peran
+                    </div>
+<svg
+                      data-pencil-name="Icon trailing"
+                      data-icon-name="chevron-down"
+                      data-icon-set="lucide"
+                      viewBox="0 0 13.99993896484375 14"
+                      preserveAspectRatio="xMidYMid meet"
+                      xmlns="http://www.w3.org/2000/svg"
+                      class="box-border w-[16px] shrink-0 h-[16px]"
+                    >
+</svg>
+</div>
+<div
+                    data-pencil-name="Message"
+                    class="box-border w-full h-fit shrink-0 flex flex-row gap-[8px] justify-start items-center"
+                  >
+<div
+                      data-pencil-name="Helper"
+                      class="text-[12px]/[17px] box-border [flex:1_1_0] text-[#71717A] font-['Plus_Jakarta_Sans',system-ui,sans-serif] font-normal text-left"
+                    >
+                      Bisa lebih dari satu. Peran baru bisa dibuat dari sini.
+                    </div>
+</div>
+</div>
+</div>
+</div>
+<div
+              data-pencil-name="Footer"
+              class="[box-sizing:content-box] w-[560px] h-[73.5px] shrink-0 flex flex-row gap-[12px] p-[16px_24px] justify-end items-center bg-[#F4F4F5] [border-width:1px_0px_0px_0px] [border-style:solid] [border-color:#F0F0F2] [margin:-0.5px_0px_0px_0px]"
+            >
+<div
+                data-pencil-name="Actions"
+                class="box-border w-fit shrink-0 h-fit flex flex-row gap-[12px] justify-start items-start"
+              >
+<div
+                  data-pencil-name="Batal"
+                  class="box-border w-fit shrink-0 h-fit flex flex-row gap-[8px] p-[12px_36px] justify-center items-center bg-[#FFFFFF] [border:1px_solid_#F0F0F2] rounded-[999px]"
+                >
+<div
+                    data-pencil-name="Label"
+                    class="text-[14px]/[normal] box-border text-[#18181B] font-['Plus_Jakarta_Sans',system-ui,sans-serif] font-semibold text-left [white-space:nowrap]"
+                  >
+                    Batal
+                  </div>
+</div>
+<div
+                  data-pencil-name="Simpan"
+                  class="box-border w-fit shrink-0 h-fit flex flex-row gap-[8px] p-[12px_36px] justify-center items-center bg-[#2F5BFF] rounded-[999px]"
+                >
+<div
+                    data-pencil-name="Label"
+                    class="text-[14px]/[normal] box-border text-[#FFFFFF] font-['Plus_Jakarta_Sans',system-ui,sans-serif] font-semibold text-left [white-space:nowrap]"
+                  >
+                    Simpan
+                  </div>
+</div>
+</div>
+</div>
+</div>
+</div>
+</div>
 </div>
 </body>
